client/web/src/enterprise/executors/secrets/backend.ts

```diff
@@ -0,0 +1,287 @@
+    query OrgExecutorSecrets($org: ID!, $scope: ExecutorSecretScope!, $first: Int, $after: String) {
+        node(id: $org) {
+            __typename
+            ... on Org {
+                executorSecrets(scope: $scope, first: $first, after: $after) {
+                    ...ExecutorSecretConnectionFields
+                }
+            }
+        }
+    }
+
+    ${EXECUTOR_SECRET_CONNECTION_FIELDS}
+`
+
+export const orgExecutorSecretsConnectionFactory = (
+    org: Scalars['ID'],
+    scope: ExecutorSecretScope
+): UseConnectionResult<ExecutorSecretFields> =>
+    // Scope has to be injected dynamically.
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    useConnection<OrgExecutorSecretsResult, OrgExecutorSecretsVariables, ExecutorSecretFields>({
+        query: ORG_EXECUTOR_SECRETS,
+        variables: {
+            org,
+            scope,
+            after: null,
+            first: 15,
+        },
+        options: {
+            fetchPolicy: 'no-cache',
+        },
+        getConnection: result => {
+            const { node } = dataOrThrowErrors(result)
+
+            if (!node) {
+                throw new Error('Org not found')
+            }
+            if (node.__typename !== 'Org') {
+                throw new Error(`Node is a ${node.__typename}, not an Org`)
+            }
+
+            return node.executorSecrets
+        },
+    })
+
+export const GLOBAL_EXECUTOR_SECRETS = gql`
+    query GlobalExecutorSecrets($scope: ExecutorSecretScope!, $first: Int, $after: String) {
+        executorSecrets(scope: $scope, first: $first, after: $after) {
+            ...ExecutorSecretConnectionFields
+        }
+    }
+
+    ${EXECUTOR_SECRET_CONNECTION_FIELDS}
+`
+
+export const globalExecutorSecretsConnectionFactory = (
+    scope: ExecutorSecretScope
+): UseConnectionResult<ExecutorSecretFields> =>
+    // Scope has to be injected dynamically.
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    useConnection<GlobalExecutorSecretsResult, GlobalExecutorSecretsVariables, ExecutorSecretFields>({
+        query: GLOBAL_EXECUTOR_SECRETS,
+        variables: {
+            after: null,
+            first: 15,
+            scope,
+        },
+        options: {
+            useURL: true,
+            fetchPolicy: 'no-cache',
+        },
+        getConnection: result => {
+            const { executorSecrets } = dataOrThrowErrors(result)
+
+            return executorSecrets
+        },
+    })
+
+export const EXECUTOR_SECRET_ACCESS_LOGS = gql`
+    query ExecutorSecretAccessLogs($secret: ID!, $first: Int, $after: String) {
+        node(id: $secret) {
+            __typename
+            ... on ExecutorSecret {
+                accessLogs(first: $first, after: $after) {
+                    ...ExecutorSecretAccessLogConnectionFields
+                }
+            }
+        }
+    }
+
+    fragment ExecutorSecretAccessLogConnectionFields on ExecutorSecretAccessLogConnection {
```

Comment:
```suggestion
    fragment ExecutorSecretAccessLogsConnectionFields on ExecutorSecretAccessLogConnection {
```

I think the GraphQL schema type should also be plural but I know that was from the last PR, sorry. 😬